spot-contracts/contracts/vaults/RolloverVault.sol

```diff
@@ -163,11 +177,12 @@ contract RolloverVault is
 
     /// @inheritdoc IVault
     /// @dev Its safer to call `recover` before `deploy` so the full available balance can be deployed.
-    ///      Reverts if there are no funds to deploy.
+    ///      Reverts if no funds are rolled over or if the minimum deployment threshold is not reached.
     function deploy() public override nonReentrant whenNotPaused {
-        TrancheData memory td = _tranche(perp.getDepositBond());
-        if (_rollover(perp, td) <= 0) {
-            revert NoDeployment();
+        (uint256 deployedAmt, TrancheData memory td) = _tranche(perp.getDepositBond());
+        uint256 perpsRolledOver = _rollover(perp, td);
+        if (deployedAmt <= minDeploymentAmt || perpsRolledOver <= 0) {
```

Comment:
Do we still need to check `perpsRolledOver <= 0`?

I think this would only be the case if we roll over a small enough dust amount of underlying such that perps rounds to 0. Would we still want to revert in that case? I think we'd want it to go through.

Reply:
Not having this check, opens up a grief attack when external callers can just tranche the underlying and the batch job has to pay for the gas to untranche (and or retranche into a new bond).

Reply:
Ah, nice catch. Basically enforces that we only tranche the underlying if it can immediately be used for rotations.

Might be worth a code comment